### PR TITLE
examples: add an example script for sending ussd codes

### PR DESCRIPTION
**Pull request by mmubarak0**
I do not know how, but sending a USSD code improves internet speed. It sometimes fixes the connection, which makes me send USSD codes frequently. other people may want to use it differently. maybe for knowing the data status more accurately than the one the UI shows. I have added some colors and waiting animation to make it visually appealing.
I'm not an experienced developer so any feedback would be helpful thanks.

### CODE REVIEW

File: examples/send_ussd.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            else:
+                print('Error: Cannot send USSD code')
+        except Exception as e:
+            print(f'Error: {e}')
+
+        # Wait for the response from the service provider.
+        while int(client.ussd.status().get('result', '1')) >= 1:
+            if wait_time >= MAX_WAIT_TIME:
+                print('Error: Timeout Limit Exceeded')
+                DONE = True
```
Comment (Salamek) [inline examples/send_ussd.py:73]: There should be a `break` here to break that while loop on line 70.

**Review by Salamek — CHANGES_REQUESTED** [review]
Anyway, code could be bit better and simpler (single thread), but it is ~ok as an example i guess so please just fix that one bug and i'll merge ;-)

File: examples/send_ussd.py (hunk 1 above)
Comment (mmubarak0) [reply examples/send_ussd.py:73]: Okay, thanks I will retake a look at the code and fix related errors if there are any.

**Comment by Salamek** [comment]
Ok, also there are some lint issues:

```
examples/send_ussd.py:38: error: Function is missing a return type annotation  [no-untyped-def]
  examples/send_ussd.py:38: note: Use "-> None" if function does not return a value
  examples/send_ussd.py:60: error: Non-overlapping equality check (left operand type: "Dict[str, Any]", right operand type: "Literal['OK']")  [comparison-overlap]
  Found 2 errors in 1 file (checked 108 source files)
```

**Comment by mmubarak0** [comment]
I did fix these errors now but I think I have done it the wrong way by running `mypy .` manually. 
what is the recommended way for running the lints checks?

**Comment by Salamek** [comment]
Running mypy . locally should do the trick, try to push those changes here and see how it goes ;-)

**Pull request merged**